gr-blocks/python/blocks/qa_file_source_sink.py

```diff
@@ -104,6 +105,67 @@ def test_file_source_can_seek_after_open(self):
 
         os.remove(filename)
 
+    def test_begin_tag(self):
+        src_data = range(1000)
+        expected_result = range(1000)
+
+        filename = "tmp.32f"
```

Comment:
I have now done what you suggested. There is a new PR for the old parts of the test here: https://github.com/gnuradio/gnuradio/pull/1311